src/matrix_input.jl

```diff
@@ -228,3 +229,81 @@ struct MILP{T, LP<:AbstractLPForm{T}}
     lp::LP
     variable_type::Vector{VariableType}
 end
+
+struct ConicForm{T, AT<:AbstractMatrix{T}} <: AbstractConicForm{T}
```

Comment:
that would be nice to add the vector in the templates of `ConicForm`, as we did in #4 . That would allow us to use `ConicForm`  in the GPU :-)

Reply:
thats a great idea. adding it now

Reply:
although i dont how should we edit the matrices A,b,c in CuArray form